zipkin/src/main/java/com/linecorp/armeria/common/tracing/RequestContextCurrentTraceContext.java

```diff
@@ -50,9 +50,20 @@
      * Use this singleton when building a {@link brave.Tracing} instance for use with
      * {@link HttpTracingService} or {@link HttpTracingClient}.
      *
+     * <p>If you need to customize the context, use {@linkplain #newBuilder()} instead.
```

Comment:
`@linkplain` -> `@link`

Reply:
addressed in the other PR (will rebase this once merged)